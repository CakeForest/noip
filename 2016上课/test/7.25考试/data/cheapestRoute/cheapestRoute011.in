13
6 0 0 0 0 0 0 
6 5 4 1 0 2 2 
6 4 1 4 3 1 4 
0
6 10 -1 10 10 -1 10 
2 0 2 
2 3 5 
7
3 1 0 -1 
1 0 
1 2 
0
23 1 2 3 4 5 6 7 8 9 234 54 234 2 123 124 3 451 32 14 987 122 321 312 
11 1 9 2 7 3 4 6 3 5 11 17 
11 8 5 3 5 3 6 3 6 7 12 14 
797
7 0 100 5 2 1 100 1 
2 0 2 
2 4 6 
0
3 1 0 -1 
1 0 
1 2 
5
3 1 1 1 
2 0 0 
2 2 1 
1
9 1 1 -1 5 6 7 8 2 3 
4 0 3 1 6 
4 3 1 6 8 
1
36 0 1 2 3 4 -1 7 8 -1 9 10 -1 34 50 54 34 44 -1 1 2 3 4 -1 5 6 7 8 9 10 -1 9 10 -1 4 -1 5 
21 0 0 0 0 7 7 7 6 6 4 10 10 12 16 16 31 31 26 28 28 33 
21 5 6 7 8 3 2 1 3 4 9 11 12 7 31 30 26 7 0 0 33 35 
55
4 1000 1000 1000 1000 
3 0 1 2 
3 2 3 1 
1
50 500 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 400 
50 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 47 
50 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 43 44 45 46 47 48 49 1 
399
50 4 2 1 0 5 6 0 3 0 5 3 5 2 3 1 34 32 12 4 2 12 3 4 3 12 31 2 12 32 21 32 1 3 22 32 4 34 2 12 32 12 32 2 12 21 21 2 23 1 23 
24 4 4 3 7 5 4 2 5 8 4 23 12 23 12 3 4 12 23 15 21 19 14 21 25 
24 7 3 5 0 5 4 5 0 8 3 12 23 12 34 32 11 23 12 23 36 18 34 28 21 
21
50 321 324 435 432 32 31 232 3 3 3 3 2 43 43 3 5 7 4 4 545 4 53 25 253 35 53 5 35 35 35 35 55 352 235 522 353 235 325 325 432 434 434 235 123 653 234 234 323 352 451 
50 12 12 12 12 12 21 1 1 21 43 43 23 23 32 32 32 32 32 32 15 14 32 12 3 1 2 3 4 4 5 6 7 8 9 10 32 32 32 32 11 11 11 11 11 11 11 11 11 11 11 
50 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 14 11 17 43 32 23 18 42 24 24 24 24 24 24 24 24 33 11 11 32 0 0 0 0 
1000
