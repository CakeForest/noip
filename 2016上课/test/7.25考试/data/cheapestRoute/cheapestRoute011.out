0 3 
25 3 
-1  -1
3057 22 
8 4 
-1  -1
1 1 
4 3 
566 17 
6 3 
19599 49 
277 29 
4501 12 
